package.json

```
@@ -4,6 +4,9 @@
   "exports": "./build/src/main.js",
   "version": "0.0.0-semantically-released",
   "description": "A pipeline for the NDE Dataset Knowledge Graph",
+  "engines": {
+    "node": ">=16"
```

20 is fine here, which is LTS now:

```suggestion
    "node": ">=20"
```